From f9bd1e183c442cc4aa2ddf99897b45cbfe62afba Mon Sep 17 00:00:00 2001
Message-ID: <f9bd1e183c442cc4aa2ddf99897b45cbfe62afba.1745272461.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1745272461.git.sharlatanus@gmail.com>
References: <cover.1745272461.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 20 Apr 2025 22:34:10 +0100
Subject: [PATCH 05/32] gnu: stellarium: Update to 25.1-1.8a3a53a.

* gnu/packages/astronomy.scm (stellarium): Update to 25.1-1.8a3a53a.
[inputs]: Add md4c.

Change-Id: I8b1732d7c385bca80ec8fea38166ab7fb5142404
---
 gnu/packages/astronomy.scm | 14 ++++++++------
 1 file changed, 8 insertions(+), 6 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index b9ab660244..02108839aa 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -70,6 +70,7 @@ (define-module (gnu packages astronomy)
   #:use-module (gnu packages linux)
   #:use-module (gnu packages lua)
   #:use-module (gnu packages machine-learning)
+  #:use-module (gnu packages markup)
   #:use-module (gnu packages maths)
   #:use-module (gnu packages multiprecision)
   #:use-module (gnu packages ncurses)
@@ -8219,14 +8220,14 @@ (define-public stackistry
     (license license:gpl3+)))
 
 (define-public stellarium
-  ;; XXX fatal error: libs/indiclient/baseclient.h: No such file or directory
-  ;; <https://github.com/Stellarium/stellarium/issues/4019>, using the latest
+  ;; XXX: 25.1 does not provide option to build with system MD4C, see
+  ;; <https://github.com/Stellarium/stellarium/issues/4267>, using the latest
   ;; commit.
-  (let ((commit "60b6becd4147dd7c731ed0be790b1a30cdc0039b")
-        (revision "0"))
+  (let ((commit "8a3a53aa5d81c6b1b2cd10546c6d40a33204091a")
+        (revision "1"))
     (package
     (name "stellarium")
-    (version (git-version "24.4" revision commit))
+    (version (git-version "25.1" revision commit))
     (source
      (origin
        (method git-fetch)
@@ -8235,7 +8236,7 @@ (define-public stellarium
              (commit commit)))
        (file-name (git-file-name name version))
        (sha256
-        (base32 "1i00f63pmyy55mwagab4pv419agzmg2c4075hc8cgbhhhfr6gm8d"))))
+        (base32 "0k9zf29x1nqj1i9qqp5mpn2z2i0izhxplyzcrlhl0i6c863rk5wx"))))
     (build-system qt-build-system)
     ;; TODO: Complete documentation build and split into dedicated outputs.
     (arguments
@@ -8261,6 +8262,7 @@ (define-public stellarium
            gpsd
            indi-2.0
            libnova
+           md4c
            nlopt
            openssl
            qtbase-5
-- 
2.49.0

